packages/table/test/formatsTests.tsx

```diff
@@ -115,7 +119,11 @@ describe("Formats", () => {
                 The fair Ophelia! -- Nymph, in thy orisons
                 Be all my sins remembered.
             `;
-            const comp = harness.mount(<TruncatedFormat truncateLength={0}>{str}</TruncatedFormat>);
+            const comp = harness.mount(
+                <div className={Classes.TABLE_NO_WRAP_TEXT}>
+                    <TruncatedFormat truncateLength={0}>{str}</TruncatedFormat>
+                </div>,
```

Comment:
Hmmm... 

So what does "TruncatedFormat" on its own mean? My reading/understanding of its behavior is that if detectTruncation is true (on by default) then essentially all it is doing is detecting whether or not it is truncated (as constrained by its parent) and if so is displaying the three-dot popover button. 

I'm going to add more tests that test the same behavior with vertical truncation as well as horizontal -- but I'm not sure it makes sense for TruncatedFormat to truncate on its own if detectTrunctation is turned on. If this is incorrect what would you rather the behavior be, here?